src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.model;
+
+public enum Fruit {
```

Comment:
can better make fruit class?

Reply:
IMHO it will be easier to extend functionality, using Enum